libs/klv/src/main/java/org/codice/alliance/libs/klv/LocationKlvProcessor.java

```diff
@@ -63,9 +65,12 @@ public GeometryOperator getGeometryFunction() {
         return postUnionGeometryOperator;
     }
 
-    private Optional<KlvHandler> find(Map<String, KlvHandler> handlers, String name) {
+    private Optional<KlvHandler> find(Map<String, KlvHandler> handlers, String name,
```

Comment:
I _think_ you can use a generic type here and not have to cast the returned value where `find` is used:

```
private <T extends KlvHandler> Optional<T> find(Map<String, T> handlers, String name, Class<T> expectedType) {
```